sdk/rust/oak/build.rs

```diff
@@ -5,7 +5,6 @@ fn main() {
             "../../../third_party/google/rpc/code.proto",
             "../../../third_party/google/rpc/status.proto",
             "../../../oak/proto/grpc_encap.proto",
-            "../../../oak/proto/oak_api.proto",
```

Comment:
Are the other proto's used or intended to be used anywhere else? If yes, perhaps we should abstract all the protos to some `oak_proto` crate

Reply:
It would also have the advantage of not requiring so many `build.rs`'s for proto generation, as the proto crate would be the only one.

Reply:
I suspect these (or maybe some FIDL equivalents) will eventually move into the `oak_abi` crate, as they're going to form part of the ABI contract: if you want to communicate with (say) the runtime's storage pseudo-Node, you're expected to exchange messages serialized according to this schema.

(But I think this is for another day &ndash; the Rust-based runtime doesn't yet support all the different pseudo-Node types.)